### PR TITLE
Add more ext tests

### LINKED ISSUE #1556: Add extensibility tests
**Description**

Add tests for the extensibility views (e2e and unit tests). 
Thought: maybe we could reuse the examples folder for the tests, to also test if examples are up-to-date?

- [x] adjust the CI pipeline
- [x] create a test for the Pizza examples
- [x] update examples/resources
- [x] ~create a test for the Certificates examples (see Pizzas)~ example has been removed
- [x] ~create a test for the Git Repos examples (see Pizzas)~ example has been removed
- [x] create a test for the Services examples (see Pizzas)

**Reasons**

Extensibility is a big feature, we should make sure it is stable.

**Attachments**

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Remove Certificates and Git repos examples (too many examples to support, it's not needed)
- Add tests for services example
- Fix crashing view, fix 'undefined' as a description

**Related issue(s)**

Resolves #1556

### CODE REVIEW

File: tests/cypress.json [hunk 1]
```diff
@@ -36,7 +35,10 @@
     "cluster/test-command-palette.spec.js",
     "cluster/test-custom-resources.spec.js",
     "cluster/test-navigation-features.spec.js",
+    "cluster/test-pizzas.spec.js",
```
Comment (mrCherry97) [inline tests/cypress.json:38]: I think you can remove that

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
after changing file organization there are problems with file path
image

**Review by mrCherry97 — DISMISSED** [review]

File: examples/services/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Resources
+
+## Overview
+
+The `services` example showcases the Busola's extensibility feature with the native Kubernetes resource.
```
Comment (grego952) [inline examples/services/README.md:5]:
```suggestion
The `services` example showcases the Busola's extensibility feature with the Kubernetes Service resource.
```

File: examples/services/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The `configuration.yaml` file consists of a ConfigMap that store the information about the extensibility configuration.
```
Comment (grego952) [inline examples/services/README.md:7]:
```suggestion
The `configuration.yaml` file consists of a ConfigMap that stores the information about the extensibility configuration.
```

**Review by grego952 — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**